packages/OmegaPrint-Core.package/CodeHolder.extension/instance/omegaPrintString.st

```diff
@@ -0,0 +1,8 @@
+*OmegaPrint-Core
+omegaPrintString
```

Comment:
Wenn man dem Methodenkommentar glaubt, macht diese Methode genau dasselbe wie `showingOmegaPrint` und sollte daher entfernt werden.

Reply:
Falls die Methode doch erforderlich ist, sollte man meiner Meinung nach an der `ifTrue:` `ifFalse` Einrückung noch Änderungen vornehmen.

Reply:
ist eine Methode die im quintupel mit benötigt wird, `showingOmegaPrint` wird in `toggleOmegaPrint `gerufen
Hier wird daraus ein String erstellt.
Also werden schon beide benötigt.

Reply:
Ok, dann nur die Einrueckung der Bloecke